apps/web/src/app/secrets-manager/secrets-manager-landing/sm-landing-api.service.ts

```diff
@@ -0,0 +1,16 @@
+import { Injectable } from "@angular/core";
+
+import { ApiService } from "@bitwarden/common/abstractions/api.service";
+
+import { RequestSMAccessRequest } from "../models/requests/request-sm-access.request";
+
+@Injectable({
+  providedIn: "root",
+})
+export class SmLandingApiService {
+  constructor(private apiService: ApiService) {}
+
+  async requestSMAccessFromAdmins(request: RequestSMAccessRequest): Promise<any> {
+    return this.apiService.send("POST", "/request-access/request-sm-access", request, true, false);
+  }
```

Comment:
```suggestion
  async requestSMAccessFromAdmins(request: RequestSMAccessRequest): Promise<void> {
    await this.apiService.send("POST", "/request-access/request-sm-access", request, true, false);
  }
```
Doesn't look like we expect this to return anything.

Reply:
@Thomas-Avery even though it's optional, I made this change because it would be weird and confusing for someone in the future if they saw this and thought it would be returning something 😆